File: solutions/python/3778/main.py
import sys

input = lambda: sys.stdin.readline().rstrip()
T = int(input())
for tc in range(T):
    S, T = input(), input()
    cntS, cntT = {}, {}
    for ch in range(ord("a"), ord("z") + 1):
        cntS[chr(ch)] = 0
    for ch in S:
        cntS[ch] = cntS.get(ch, 0) + 1
    for ch in T:
        cntT[ch] = cntT.get(ch, 0) + 1

    answer = 0
    for k, valueS in cntS.items():
        valueT = cntT.get(k, 0)
        answer += valueS + valueT - 2 * min(valueS, valueT)
    print(f"Case #{tc+1}: {answer}")

File: solutions/cpp/3778/main.cpp
#include<bits/stdc++.h>

using namespace std;

int main() {
    ios::sync_with_stdio(false);
    cin.tie(0);

    int T; cin >> T;
    cin.ignore();
    for (int tc = 1; tc <= T; ++tc) {
        string S; getline(cin, S);
        string T; getline(cin, T);
        vector<int> cntS(26, 0), cntT(26, 0);
        for (char ch : S) ++cntS[ch - 'a'];
        for (char ch : T) ++cntT[ch - 'a'];
        int answer = 0;
        for (int i = 0; i < 26; ++i) {
            answer += cntS[i] + cntT[i] - 2 * min(cntS[i], cntT[i]);
        }
        cout << "Case #" << tc << ": " << answer << '\n';
    }
    return 0;
}
